components/Balances.tsx

```diff
@@ -0,0 +1,34 @@
+import { useBalances } from '@/components/hooks';
+import type { ValueView } from '@penumbra-zone/protobuf/penumbra/core/asset/v1/asset_pb';
+import { ValueViewComponent } from '@penumbra-zone/ui/ValueViewComponent';
+import type React from 'react';
+
+export function Balances() {
+  const { data: balances } = useBalances();
+
+  return balances
+    ? balances
+        .map((bal) => bal.balanceView)
+        .map((balanceView) => (
```

Comment:
Not a big deal, but you can avoid the first loop by doing:
```suggestion
        .map(({ balanceView }) => (
```